rest-api/config/config_prod.json

```diff
@@ -17,8 +17,14 @@
     },
     "exporter@all-of-us-rdr-prod.iam.gserviceaccount.com": {
       "roles": ["exporter"]
-    }
-  },
+    },
+    "awardee-tach@all-of-us-ops-data-api-stable.iam.gserviceaccount.com": {
+      "roles": ["awardee_sa"],
+      "awardee": "TRANS_AM",
+      "whitelisted_ip_ranges" : {
+        "ip4": ["66.189.127/27", "68.124.42.162/27"]
```

Comment:
You don't need a fourth part of 66.189.127?

Reply:
No, it will cover .0-.32. It's redundant.